oar/cli/cmd_update_bug_list.py

```diff
@@ -16,12 +16,16 @@
 @click.option("--notify/--no-notify", default=True, help="Send notification to bug owners, default value is true")
 @click.option("--confirm-droppable", is_flag=True, default=False,
               help="Send notification only to bug owners with critical and higher issue severity, default value is false")
-def update_bug_list(ctx, notify, confirm_droppable):
+@click.option("--notify-team-leads", is_flag=True, default=False, help="Send notification to team leads of unverified CVE issues, default value is false")
+@click.option("--notify-managers", is_flag=True, default=False, help="Send notification to managers of unverified CVE issues, default value is false")
+def update_bug_list(ctx, notify, confirm_droppable, notify_team_leads, notify_managers):
     """
     Update bug status listed in report, update existing bug status and append new ON_QA bug
     """
-    if not notify and confirm_droppable:
-        raise click.UsageError("Error: --no-notify and --confirm-droppable cannot be used together")
+    if not notify and (confirm_droppable or notify_team_leads or notify_managers):
+        raise click.UsageError("Error: --no-notify cannot be used together with --confirm-droppable, --notify-team-leads or --notify-managers")
+    if sum([confirm_droppable, notify_team_leads, notify_managers]) > 1:
+        raise click.UsageError("Error: only one of parameters --confirm-droppable, --notify-team-leads or --notify-managers can be used simultaneously")
```

Comment:
@LuboTerifaj Please take a look. I updated the checks:

1. `--no-notify` cannot be used with any of `--confirm_droppable` or `--notify_team_leads` or `--notify_managers`
2. Only one of `--confirm_droppable` or `--notify_team_leads`, `--notify_managers` can be used simultaneously

Reply:
Thanks!